install_config/install/advanced_install.adoc

```diff
@@ -1311,18 +1347,16 @@ node1.example.com           Ready                      165d
 node2.example.com           Ready                      165d
 ----
 
-To verify that the web console is installed correctly, use the master host name and the console port number to access the console with a web browser.
-
-For example, for a master host with a hostname of `master.openshift.com` and using the default port of `8443`, the web console would be found at:
-
-----
-https://master.openshift.com:8443/console
-----
+. To verify that the web console is installed correctly, use the master host name
+and the web console port number to access the web console with a web browser.
++
+For example, for a master host with a host name of `master.openshift.com` and
+using the default port of `8443`, the web console would be found at `\https://master.openshift.com:8443/console`.
```

Comment:
As an aside, this information is hard to find. Should we add that the console IP is the master host name and the web console port number in other locations? For example:
https://docs.openshift.com/container-platform/3.5/architecture/infrastructure_components/web_console.html